compiler/vm/vmdef.nim

```diff
@@ -410,13 +429,44 @@ type
                                   ## symbol identifier
   VmCallback* = proc (args: VmArgs) {.closure.}
 
+  VmCell* = object
+    ## Stores information about a memory cell allocated by the VM's allocator
+    count*: int
```

Comment:
Not necessary now, just a thought; I'd favour hiding the field behind accessors.

```suggestion
    countOrNextPtr*: int
```

Reply:
Yep, it's indeed better. Since the accessors would have to be defined in `vmdef` for now (where they don't really belong), I left a to-do in the code for adding the accessor routines once `VmCell` is owned by the `vmmemory` module.